internal_worker.go

```diff
@@ -748,7 +760,9 @@ func (aw *aggregatedWorker) Stop() {
 	aw.logger.Info("Stopped Worker")
 }
 
-// aggregatedWorker returns an instance to manage the workers.
+// aggregatedWorker returns an instance to manage the workers. Use defaultConcurrentPollRoutineSize (which is 1) as
+// poller size. The typical RTT (round-trip time) is below 1ms within data center. With 1 poller, we could easily achieve
```

Comment:
more interesting is the median poll API latency, which based on our dashboards is ~5-6ms..